cookbooks/tomography_based_plate_motions/plugins/tomography_based_plate_motions.cc

```diff
@@ -1350,6 +1528,23 @@ namespace aspect
                              "We can evaluate the effects of lithospheric thickness variations on the surface plate "
                              "motions using this. Currently, this parameter sets the lithosphere to a constant "
                              "thickness of 100 km.");
+          prm.declare_entry ("Use slab2 database", "false",
+                             Patterns::Bool (),
+                             "This parameter value determines if we want to the slab depths and thickness defined "
+                             "in Slab2 database (Hayes et al., 2018). Currently, it only supports input as a 2D   "
+                             "grid on the boundary with columns representing depths and thickness of the slabs.");
+          prm.declare_entry ("Slab temperature anomaly", "-650",
+                             Patterns::Double(),
+                             "The temperature anomaly of the slabs relative to the surrounding mantle. Since we use the diffused "
+                             "temperature anomaly in our current implementation, the default value is approximately the difference "
```

Comment:
```suggestion
                             "The temperature anomaly of the slabs relative to the surrounding mantle. Since we diffuse the slab "
                             "temperature anomaly from this initial value, the default value is approximately the difference "
```